.github/workflows/github-release.yml

```diff
@@ -1,8 +1,8 @@
-name: Release
+name: Release Webhook Listener
 
 on:
   push:
-    branches: [master]
+    branches: [main]
```

Comment:
@cs-raj We use `master` instead of `main` in webhook-listener. In order to avoid issues upon release, would you kindly update and verify to see if everything is functioning as it should?

Reply:
Done @aman19K